security-advisories/security-advisories.md

```diff
@@ -18,6 +18,76 @@ If you have found a security issue in OP-TEE, please send us an email (see
 About) and then someone from the team will contact you for further discussion.
 The initial email doesn't have to contain any details.
 
+# January 2018
+## Meltdown and Spectre
+In collaboration a group of different people (see "Reported by") have found out
+that it is possible to circumvent security countermeasures and privilege
+escalation by using speculative execution, caches, out-of-order execution in a
+specific way. All details about the attacks has been thoroughly described in
+the whitepapers that can found in the [Meltdown and Spectre] page. So we will
+not cover the details here, instead we will highlight how it could affect
+OP-TEE and what the current status is about the mitigations.
+
+Note that since there are some patches still in progress, we will update this
+page when things have changed.
+
+### Variant 1: bounds check bypass (CVE-2017-5753)
+
+#### Possible attack
+Could be that an attacker finds a gadget that can be used as a trampoline to
+get access kernel memory from a Trusted Application for example. Since user
+data provided to Trusted Applications most often comes from non-secure side it
+is important to check the code where we are using non-secure parameters as well
+as when doing syscalls from Trusted Applications. 
+
+#### Current status:
+Still under investigation, we have received GCC patches from Arm that can help us finding vulnerable areas.
+
+| Reported by  | CVE ID | OP-TEE ID | Affected versions |
+| ------------ |:------:| :-------: | ----------------- |
+| [Google Project Zero], [University of Pennsylvania], [University of Maryland], [Rambus], [Graz University of Technology], [University of Adelaide], [Data61] | [CVE-2017-5753](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-5753) | OP-TEE-2018-0001 | All versions prior to `OP-TEE 3.0.0` (not yet released) |
+
+### Variant 2: branch target injection (CVE-2017-5715)
```

Comment:
This section is slightly incomplete/incorrect, should probably differentiate between:
1. context switches from nsec to sec on the one hand,
2. secure world exceptions/syscalls on the other hand, causing transition from SEL0 to SEL1

Arm-TF 1214 and OP-TEE 2047 address (1) while OP-TEE 2055 (32-bits) and OP-TEE TBD (64-bits) address (2).